contrib/survey_cake/udfs/survey_cake_csv_uploader.py

```diff
@@ -97,27 +125,23 @@ def _upload_2_bigquery(self, file_path, table_id):
 
         table = self.client.get_table(table_id)  # Make an API request.
         print(
-            f"Loaded {table.num_rows} rows and {len(table.schema)} columns to {table_id}"
+            f"There's {table.num_rows} rows and {len(table.schema)} columns in {table_id} now!"
         )
 
     def _generate_question_id_dimension_table(self, header):
+        max_existing_question_id = int(max(self.existing_question_and_id_dict.values()))
         question_id_dim_table = {}
-        for index, column in enumerate(header):
+        for index, column in enumerate(header, start=max_existing_question_id):
```

Comment:
this is how I auto-increment question id
1. get the max question id
2. use `enumerate